### ISSUE TITLE
Genbank file is not optional

### ISSUE THREAD

**Issue by mthang** [bgruening/galaxytools#1389]
It is misleading to see the genbank is labeled as optional (line 100) in this wrapper. Line 21 tells otherwise if I am not mistaken.
https://github.com/bgruening/galaxytools/blob/af8605d266717ed3453bd2a0947fed79c7098fb3/tools/mitohifi/mitohifi.xml#L21

**Comment by bgruening** [comment]
True, but which one is correct?

**Comment by mthang** [comment]
My colleague and I tested the MitoHiFI with genbank file provided , and it works. The tool fails without the genbank file. It looks like the genbank file is a requirement for MitoHiFI according to this git repo https://github.com/marcelauliano/MitoHiFi

**Comment by bgruening** [comment]
Ok, cool, do you remove the `optional="true"` or should I?

**Comment by mthang** [comment]
Yes. remove the optional="true" at your end. I also spotted the multiple="true" in the wrapper for the genbank file input and I dont know if the MitoHiFI could take a list of genbank file as an input.
